### DISCUSSION TITLE
Services that will verify throttling levels?

### DISCUSSION THREAD

**Discussion by philCryoport** [kickstarter/rack-attack#676] [Questions (Q&A)]
Hi folks, we're a life sciences company -- and thus, if we set a throttle level limit for production servers, we should verify that throttle level limit in the validation environment.

I know we could roll-our-own with JMeter or Postman -- but that also means setting up significant infrastructure to verify "no throttling below X; throttling at X or above".  

Rather than re-invent the wheel, it would be great if we could pay for a service that we can configure and push a button -- and thus shown the throttling only occurs at a level of X or higher.

Googling hasn't found anything yet. Any suggestions / recommendations for SaaS's?

**Comment by mpenalozag** [comment]
@philCryoport Hey Phil! I got a bit late for this question. Have you tried tools like [apache bench](https://httpd.apache.org/docs/2.4/programs/ab.html) to send requests to your webservers? Is this something that could work for you or do you need something more elaborated or something else entirely?
